flow/alerting/classifier.go

```diff
@@ -296,6 +295,9 @@ func GetErrorClass(ctx context.Context, err error) (ErrorClass, ErrorInfo) {
 			1195, // ER_CRASHED_ON_REPAIR
 			1827: // ER_PASSWORD_FORMAT
 			return ErrorNotifyConnectivity, myErrorInfo
+		case 1236, // ER_MASTER_FATAL_ERROR_READING_BINLOG
+			1373: // ER_UNKNOWN_TARGET_BINLOG
```

Comment:
nit: are you aware of when this can happen?

Reply:
mariadb description is "Target log not found in binlog index" so I'm guessing it'll come up with binlog corruption or in certain failover situations